internal/config/autoconfig/autoconfig.go

```diff
@@ -210,22 +210,26 @@ func getProjectFilters(c *config.Config) ([]project.Filter, error) {
 }
 
 func getRootConfig(cfgLoader *config.Loader, userCfgDir UserConfigDir) (*config.Config, error) {
+	var cfg *config.Config
+
 	content, err := cfgLoader.RootConfig()
-	if err != nil {
+	switch err {
+	case nil:
+		if cfg, err = config.ParseYAML(content); err != nil {
+			return nil, err
+		}
+	case config.ErrRootConfigNotFound:
+		cfg = &config.ConfigDefaults
```

Comment:
Let's make sure we return a copy and not a pointer to a default object.

```suggestion
		temp := config.ConfigDefaults
		cfg = &temp
```

Reply:
Made it so that `config.Defaults()` always returns a deep copy